src/App.js

```diff
@@ -183,7 +183,7 @@ class OldApp extends Component {
   }
 
   componentWillUnmount() {
-    AppState.removeEventListener('change', this.handleAppStateChange);
+    AppState?.removeEventListener('change', this?.handleAppStateChange);
```

Comment:
is it possible that `this` is undefined? just wondering bc of the optional chaining. i'm guessing it's not possible, but if it is we should add the optional to the lines below as well